CONTRIBUTING.adoc

```diff
@@ -55,8 +55,6 @@ When you make changes to the XSD Schema, it is your responsibility to ensure tha
 
 To re-generate the images, follow the steps:
 
-. Download the latest XSDDiagram binaries from https://github.com/clagms/xsddiagram/releases.
-. Unzip and place the binaries in the folder `<fmi-standard>/bin/xsdiagram` so that, e.g., the path of the binary `XSDDiagramConsole.exe` is `<fmi-standard>/bin/xsdiagram/XSDDiagramConsole.exe`
 . On a powershell terminal placed at directory `<fmi-standard>`, run `.\bin\generate_figures.ps1`.
```

Comment:
"placed at directory" no longer is necessary as the script works from any directory now.

Reply:
Simply:

Run `PowerShell -Command "path\to\generate_figures.ps1"`.

Reply:
Thank you @beutlich . Fixed now!